packages/ubuntu_bootstrap/lib/pages/storage/storage_dialogs.dart

```diff
@@ -148,10 +148,7 @@ class TpmOption extends StatelessWidget {
       compatible = false;
 
       final color = Theme.of(context).disabledColor.toHex();
-
-      data = message == null
-          ? '<span style="color: $color">Info: Unknown</span> '
-          : '<span style="color: $color">$reason: $message</span>';
+      data = '<span style="color: $color">$message</span>';
```

Comment:
Should this rather be `${message ?? ''}`?

Reply:
And could you remove the `final reason = ..` above to appease the linter? :)

Reply:
It would be nice to have a fallback message incase its empty ... but lets defer for the moment